src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/ElectrifiedCrucible.java

```diff
@@ -29,6 +31,13 @@ protected void registerDefaultRecipes() {
         for (Material leaves : Tag.LEAVES.getValues()) {
             registerRecipe(10, new ItemStack[] { new ItemStack(Material.BUCKET), new ItemStack(leaves, 16) }, new ItemStack[] { new ItemStack(Material.WATER_BUCKET) });
         }
+
+        if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
+            registerRecipe(10, new ItemStack[] {new ItemStack(Material.BUCKET),
+                new ItemStack(Material.BLACKSTONE, 8)}, new ItemStack[] {new ItemStack(Material.LAVA_BUCKET)});
+            registerRecipe(10, new ItemStack[] {new ItemStack(Material.BUCKET), new ItemStack(Material.BASALT, 8)},
+                new ItemStack[] {new ItemStack(Material.LAVA_BUCKET)});
```

Comment:
Formatting...
```suggestion
            registerRecipe(10, new ItemStack[] {new ItemStack(Material.BUCKET), new ItemStack(Material.BLACKSTONE, 8)}, new ItemStack[] {new ItemStack(Material.LAVA_BUCKET)});
            registerRecipe(10, new ItemStack[] {new ItemStack(Material.BUCKET), new ItemStack(Material.BASALT, 8)}, new ItemStack[] {new ItemStack(Material.LAVA_BUCKET)});
```

Reply:
I have no clue what happened here, I think copy pasting from GH may not have been the best idea